packages/mask/src/social-network/ui.ts

```diff
@@ -86,7 +86,10 @@ export async function activateSocialNetworkUIInner(ui_deferred: SocialNetworkUI.
     startPostListener()
     ui.collecting.currentVisitingIdentityProvider?.start(signal)
     ui.injection.pageInspector?.(signal)
-    if (Flags.toolbox_enabled) ui.injection.toolbox?.(signal)
+    if (Flags.toolbox_enabled) {
+        ui.injection.toolbox?.(signal, 'wallet')
```

Comment:
why not start both in the toolbox implementation?

Reply:
injectToolboxAtFacebook

Reply:
Toolbox implementations of SNS have already included `injectToolboxHintAtMinds`, `injectToolboxAtFacebook` and `injectToolboxHintAtTwitter`, and maybe more in the future. I want to keep dry.